### DISCUSSION TITLE
Is there a limit to the number of sites that can be monitored?

### DISCUSSION THREAD

**Discussion by cweagans** [upptime/upptime#235] [Q&A] [answered]
We manage a fleet of ~300 websites. We'd like to monitor them all. Will this work?

**Comment by AnandChowdhary** [comment, marked as answer]
No, there is no limit to how many websites you can monitor. That being said, the GitHub API apparently only allows sending a couple MB of data in one API request, so someone pointed out a problem. I say you go ahead and see if anything breaks for you! 😉
